target/bin/restrict-access

```diff
@@ -44,7 +44,8 @@ case "${MODE}" in
           sed -i 's|smtpd_sender_restrictions =|smtpd_sender_restrictions = check_sender_access texthash:/tmp/docker-mailserver/postfix-send-access.cf,|' /etc/postfix/main.cf \
         || sed -i 's|smtpd_recipient_restrictions =|smtpd_recipient_restrictions = check_recipient_access texthash:/tmp/docker-mailserver/postfix-receive-access.cf,|' /etc/postfix/main.cf
 
-      service postfix reload >/dev/null
+      _adjust_mtime_for_postfix_maincf
+      postfix reload
```

Comment:
```suggestion
      _reload_postfix
```

if `_reload_postfix` function is moved from `check-for-changes.sh` to `utils.sh`, the function can be used here.